binary_gentoo/internal/cli/build.py

```diff
@@ -176,19 +182,27 @@ def build(config):
         f'LDFLAGS={shlex.quote(config.ldflags)}',
     ]
 
+    if config.tag_docker_image is not None:
+        # TODO there should probably be some sanity checks on the provided image name here
+        container_name = f'binary-gentoo-{uuid.uuid4().hex}'
+    else:
+        container_name = None
+
     emerge = ['env'] + emerge_env + ['emerge'] + emerge_args
     emerge_quoted_flat = ' '.join(emerge)
     rebuild_or_not = f'--usepkg={"n" if config.enforce_rebuild else "y"}'
 
     container_profile_dir = os.path.join(container_portdir, 'profiles', config.gentoo_profile)
+    inside_container_portdir = '/var/db/repos/gentoo'
+    inside_container_make_profile = '/etc/make.profile'
```

Comment:
Can we rename those from `inside_container_` to just `container_` for now to match that existing naming scheme (e.g. `container_portdir` and `container_profile_dir`)? We can change the scheme to `inside_container_` everywhere some time after, if you like.

Reply:
Yes indeed, somehow I made a bad connection in my mind, thinking that the `container_` variables refer to directories on the host. But that is indeed not the case. I will rename to keep it coherent.